src/backend/configs/caliopen-go-lmtp_dev.yaml

```diff
@@ -24,18 +24,18 @@ AppConfig:
 ## LDA (Email broker) config ##
 LDAConfig:
   broker_type: smtp # types are : smtp, imap, mailboxe, etc.
-  nats_url: nats://localhost:4222
+  nats_url: nats://nats.dev.caliopen.org:4222
```

Comment:
this config shouldn't be restricted to a docker environment ?

I mean this is used for both native & vagrant env, right ?